public/app/features/dashboard/state/DashboardModel.ts

```diff
@@ -85,6 +85,7 @@ export class DashboardModel {
 
   // repeat process cycles
   iteration?: number;
+  // @ts-ignore has no initializer and is not definitely assigned in the constructor. This is assigned in the constructor on this.initMeta(meta);
   meta: DashboardMeta;
```

Comment:
think for cases like this we can use `declare meta: DashboardMeta`

Reply:
Aha brilliant suggestion, didn't know this!